specs/user_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative "test_helper"
+
+describe "User class" do
+  describe "self.list" do
+    it "can return all users" do
+      VCR.use_cassette("slack_user") do
+        response = User.list
+        user = response.first
+
+        expect(response).wont_be_nil
+        expect(response.first.name).must_equal user.name
```

Comment:
The expectation on line 11 is not useful, you've essentially said `expect(response.first.name).must_equal response.first.name`, which will always be true.